foxpuppet/ui/browser/window.py

```diff
@@ -0,0 +1,90 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from .tabbar import TabBar
+from .navbar import Navbar
+from foxpuppet.ui.windows import Windows
+from foxpuppet.ui.base_window import BaseWindow
+
+
+class Browser_Window(object):
```

Comment:
What other types of window are you referring to? I know that 'window' can refer to many things, but I imagine a user of FoxPuppet would probably only use it to refer to the browser window.